src/libcore/num/wrapping.rs

```diff
@@ -88,6 +88,15 @@ macro_rules! wrapping_impl {
             }
         }
 
+        #[cfg(not(stage0))]
+        #[unstable(feature = "op_assign_traits", reason = "recently added", issue = "28235")]
```

Comment:
@petrochenkov hmm... They don't seem to be for anything else in core::num... for example, the impl of  `Zero` is unstable.